### PR TITLE
Remove unused modules and variables

### PR DESCRIPTION
**Pull request by AlexKurek**

### CODE REVIEW

File: lib/healpy/pixelfunc.py [hunk 1]
```diff
@@ -1643,7 +1643,7 @@ def remove_dipole(
     --------
     fit_dipole, fit_monopole, remove_monopole
     """
-    input_ma = is_ma(m)
+    # input_ma = is_ma(m)
```
Comment (lpsinger) [inline lib/healpy/pixelfunc.py:1646]: Why is this commented out?

File: lib/healpy/zoomtool.py [hunk 2]
```diff
@@ -442,7 +442,7 @@ def _reso_on_key(self, ev):
             self.save_min = self._gnom_ax.images[0].norm.vmin
             self.save_max = self._gnom_ax.images[0].norm.vmax
         elif ev.key == "g":
-            if hasattr(self, "_graton") and self._graton == True:
+            if hasattr(self, "_graton") and self._graton:
```
Comment (lpsinger) [inline lib/healpy/zoomtool.py:445]:
```suggestion
            if getattr(self, "_graton", False):
```

File: test/test_sphtfunc.py [hunk 3]
```diff
@@ -184,7 +184,7 @@ def test_gauss_beam(self):
         np.testing.assert_allclose(idl_gauss_beam, gauss_beam)
 
     def test_alm2cl(self):
-        nside = 32
+        # nside = 32
```
Comment (lpsinger) [inline test/test_sphtfunc.py:187]: Why is this commented out?

File: test/test_sphtfunc.py [hunk 4]
```diff
@@ -652,8 +652,8 @@ def test_blm_gauss(self):
 def test_resize_alm(lmax, mmax, lmax_out, mmax_out):
     alm = np.random.uniform(size=hp.Alm.getsize(lmax, mmax)).astype(np.complex128)
     alm_out = hp.resize_alm(alm, lmax, mmax, lmax_out, mmax_out)
-    lmaxmax = max(lmax, lmax_out)
-    lmaxmin = min(lmax, lmax_out)
+    # lmaxmax = max(lmax, lmax_out)
+    # lmaxmin = min(lmax, lmax_out)
```
Comment (lpsinger) [inline test/test_sphtfunc.py:656]: Why are these commented out?

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please update the title of this PR, as it is no longer just removing unused modules.

File: test/test_sphtfunc.py (hunk 3 above)
Comment (AlexKurek) [reply test/test_sphtfunc.py]: Removed

File: lib/healpy/pixelfunc.py (hunk 1 above)
Comment (AlexKurek) [reply lib/healpy/pixelfunc.py]: Removed.

File: test/test_sphtfunc.py (hunk 4 above)
Comment (AlexKurek) [reply test/test_sphtfunc.py]: Removed.

**Comment by AlexKurek** [comment]
Applied suggestions in https://github.com/healpy/healpy/pull/958/commits/91e1ef1d276edf0ee6ad33c50f7adcb4757d2253

**Review by lpsinger — CHANGES_REQUESTED** [review]
Readthedocs build is failing. Please fix.

**Comment by lpsinger** [comment]
The readthedocs build is still failing.

**Review by lpsinger — APPROVED** [review]
LGTM assuming the CI pipeline succeeds. Over to you, @zonca.

**Review by zonca — APPROVED** [review]

**Pull request merged**